ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+
+
+def main():
+    # Check command line arguments
+    if len(sys.argv) != 6:
```

Comment:
we might want to use `argparse` here now instead rather than all the `sys.argv` values

Reply:
Replace `sys.argv` with `argparse`.  Done at 75e5b9d.